libraries/SITL/SIM_GPS.cpp

```diff
@@ -550,19 +550,16 @@ void GPS::update_nmea(const struct gps_data *d)
                      d->have_lock?_sitl->gps_numsats[instance]:3,
                      1.2,
                      d->altitude);
-    const float speed_mps = norm(d->speedN, d->speedE);
+                     
+    const float speed_mps = d->speed_2d();
     const float speed_knots = speed_mps * M_PER_SEC_TO_KNOTS;
-
-    float heading = ToDeg(atan2f(d->speedE, d->speedN));
```

Comment:
This was a bug, I dropped the degrees. Autotest doesn't yet check heading. Issuing a patch shortly.

Reply:
https://github.com/ArduPilot/ardupilot/pull/27047